admin-http-gateway/src/main.rs

```diff
@@ -99,10 +99,11 @@ fn set_rust_log(
     state: &rocket::State<State>,
     form: Form<SetRustLogForm>,
 ) -> Result<Redirect, String> {
-    let mut req = admin::SetRustLogRequest::new();
-    req.set_rust_log(form.rust_log.clone());
+    let req = admin::SetRustLogRequest {
+        rust_log: form.rust_log.clone(),
+    };
```

Comment:
For most of the instances that were straight forward, I used the structure initializer
If one tries to do the following clippy will bark saying to use the struct initializer approach
```rust
let mut req = admin::SetRustLogRequest::default();
req.rust_log = from.rust_log.clone();
```